### PR TITLE
Feature/bump sdk version 2.1.0

### PR DESCRIPTION
**Pull request by dlwcz**
* Bump SDK version to 2.1.0

* Create applicant through API outside of SDK

* Start flow using `withApplicant(String id)`

* Delete unused `ActivityUtils.java`

### CODE REVIEW

File: app/src/main/java/com/onfido/android/app/sample/MainActivity.java [hunk 1]
```diff
@@ -1,21 +1,32 @@
...
 import com.onfido.api.client.data.Applicant;
 
+import org.json.JSONException;
+import org.json.JSONObject;
+
 public class MainActivity extends BaseActivity {
 
     private Onfido client;
+    private Activity activity = this;
+    private String applicantId = "";
```
Comment (rfreitas) [inline app/src/main/java/com/onfido/android/app/sample/MainActivity.java]: necessary to be a prop?

File: app/src/main/java/com/onfido/android/app/sample/ActivityUtils.java [hunk 2]
```diff
@@ -1,38 +0,0 @@
-package com.onfido.android.app.sample;
```
Comment (sergiomarqmoura) [inline app/src/main/java/com/onfido/android/app/sample/ActivityUtils.java:1]: Nice catch!

File: app/src/main/java/com/onfido/android/app/sample/MainActivity.java [hunk 3]
```diff
@@ -1,21 +1,32 @@
...
+import com.onfido.android.sdk.capture.utils.OnfidoApiUtil;
 import com.onfido.api.client.data.Applicant;
 
+import org.json.JSONException;
+import org.json.JSONObject;
+
 public class MainActivity extends BaseActivity {
 
     private Onfido client;
+    private Activity activity = this;
```
Comment (rfreitas) [inline app/src/main/java/com/onfido/android/app/sample/MainActivity.java]: unusual pattern. Inside of callbacks the common pattern is to do `MainActivity.this`.

**Review by sergiomarqmoura — APPROVED** [review]

File: app/src/main/java/com/onfido/android/app/sample/MainActivity.java (hunk 1 above)
Comment (sergiomarqmoura) [reply app/src/main/java/com/onfido/android/app/sample/MainActivity.java]: We need to use it inside other callback, the one triggered once the user finishes the flow (talking about the `applicantId`)

File: app/src/main/java/com/onfido/android/app/sample/MainActivity.java (hunk 3 above)
Comment (sergiomarqmoura) [reply app/src/main/java/com/onfido/android/app/sample/MainActivity.java]: 👍

**Comment by sergiomarqmoura** [comment]
@dawiddylowicz @rfreitas also, any concern about using the same applicant name as on iOS? Concretely, `Theresa May`

**Comment by dlwcz** [comment]
@sergiomarquesmoura no concern about using the same applicant name as on iOS

File: app/build.gradle [hunk 4]
```diff
@@ -11,7 +11,7 @@ final ERROR_MESSAGE = "You have not defined an app key for the Onfido api."
 
 def apiKey = getValueFromFileOrEnv(APP_TOKEN_ENV_VAR_NAME, APP_TOKEN_FILE, APP_TOKEN_KEY, ERROR_MESSAGE)
 
-def sdkVersion = '2.0.0'
+def sdkVersion = '2.1.0-rc.1'
```
Comment (dlwcz) [inline app/build.gradle]: to be changed to 2.1.0 before merging

**Review by rfreitas — APPROVED** [review]
approved with minor

File: app/src/main/java/com/onfido/android/app/sample/MainActivity.java [hunk 5]
```diff
@@ -5,17 +5,25 @@
...
 import com.onfido.android.sdk.capture.upload.Captures;
 import com.onfido.api.client.data.Applicant;
 
+import org.json.JSONException;
+import org.json.JSONObject;
+
 public class MainActivity extends BaseActivity {
 
     private Onfido client;
+    private String applicantId = "";
```
Comment (rfreitas) [inline app/src/main/java/com/onfido/android/app/sample/MainActivity.java]: Minor: seems like it doesn't need to be a property

**Pull request merged**